website/docs/docs/dbt-cloud-apis/user-tokens.md

```diff
@@ -4,19 +4,72 @@ id: "user-tokens"
 pagination_next: "docs/dbt-cloud-apis/service-tokens"
 ---
 
+:::note Action required
+
+The [user API tokens](#user-tokens) will eventually be deprecated. The deprecation date is yet to be determined, but we recommend you update to account-scoped personal access tokens to avoid service disruptions in the future. We will communicate, with ample notice, the deprecation date when it has been determined.
```

Comment:
This note is kind of wordy and could use some fine-tuning. I worry people will miss the important bits.